Assets/MRTK/Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -79,16 +79,28 @@ public MixedRealityInteractionMapping(uint id, string description, AxisType axis
 
         public MixedRealityInteractionMapping(MixedRealityInteractionMapping mixedRealityInteractionMapping)
             : this(mixedRealityInteractionMapping.id,
-                   mixedRealityInteractionMapping.Description,
-                   mixedRealityInteractionMapping.AxisType,
-                   mixedRealityInteractionMapping.InputType,
-                   mixedRealityInteractionMapping.MixedRealityInputAction,
+                   mixedRealityInteractionMapping.description,
```

Comment:
why the casing change?

Reply:
Changes from calling the property to the backing field. Not a major change, but they were inconsistent in this call (some called the property, others the field (like the above `.id`) and I chose to converge on the backing field.